hack/pretty-print-godeps.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type Godeps struct {
```

Comment:
Do you actually need this? Isn't it way safer to just unmarshal into map[string]interface?

Reply:
Tried that. It changes the order. The struct marshaler seems to go field by field.

Reply:
with @rmmh's unexpand snipped we can drop this file completely 👍